pt-BR/mail/chrome/messenger/msgHdrViewOverlay.dtd

```diff
@@ -35,34 +35,34 @@
 
  ***** END LICENSE BLOCK ***** -->
 
-<!ENTITY pbToField.label "To:">
-<!ENTITY pbFromField.label "From:">
-<!ENTITY pbSenderField.label "Sender:">
-<!ENTITY pbOrganizationField.label "Organization:">
-<!ENTITY pbReplyToField.label "Reply-To:">
+<!ENTITY pbToField.label "Para:">
+<!ENTITY pbFromField.label "De:">
+<!ENTITY pbSenderField.label "Remetente:">
+<!ENTITY pbOrganizationField.label "Organização:">
+<!ENTITY pbReplyToField.label "Responder para:">
 
-<!ENTITY subjectField.label "Subject: ">
+<!ENTITY subjectField.label "Assunto: ">
 <!--# LOCALIZATION NOTE (ccField.label): DONT_TRANSLATE -->
 <!ENTITY pbCcField.label "Cc:">
-<!ENTITY pbBccField.label "Bcc:">
-<!ENTITY pbNewsgroupsField.label "Newsgroups:">
-<!ENTITY pbFollowupToField.label "Followup-To:">
+<!ENTITY pbBccField.label "Cco:">
+<!ENTITY pbNewsgroupsField.label "Grupos de mensagens:">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pbNewsgroupsField.label "Grupos de notícias:">